test/gtest/uct/uct_test.cc

```diff
@@ -1287,7 +1263,7 @@ void uct_test::entity::connect(unsigned index, entity& other, unsigned other_ind
     }
 }
 
-void uct_test::entity::listen(const ucs::sock_addr_storage &listen_addr,
+void uct_test::entity::listen(ucs::sock_addr_storage &listen_addr,
```

Comment:
minor: maybe copy listen_addr to local var instead of changing it in place? could be unexpected for code which calls listen()

Reply:
then I would implement wrapper `listen_retry` with the loop to keep `m_listen_addr` consistent with actual listening address, WDYT?

Reply:
maybe even remove listen_addr param, and always use m_listen_addr ?

Reply:
`m_listen_addr` is a member of `test_uct_sockaddr` class not an `entity`